app/controllers/unsorted_tags_controller.rb

```diff
@@ -5,13 +5,17 @@ class UnsortedTagsController < ApplicationController
   before_action :check_permission_to_wrangle
 
   def index
+    authorize :wrangling if logged_in_as_admin?
```

Comment:
I think it would be good to use `view_access?` directly here instead of going via the action because `index` is one of the actions where the various wrangling related controllers have different access levels.

That way the `index?` alias can be removed and we won't have someone accidentally converting e.g. TagWranglersController to using the `index?` permission and giving more roles access than intended.